AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt

```diff
@@ -555,8 +555,10 @@ goteborg2023.com#@#.cookie-popup:not(body):not(html)
 @@||1a-garten-ammer.de/Scripts/CookieBanner.js
 ! https://github.com/AdguardTeam/AdguardFilters/issues/152356
 @@||cmp.osano.com/*/osano.js$domain=lucidchart.com
+! https://github.com/AdguardTeam/AdguardFilters/issues/168620
+! https://github.com/AdguardTeam/AdguardFilters/issues/168622
 ! https://github.com/AdguardTeam/AdguardFilters/issues/152345
-capital.it#@##iubenda-cs-banner
+m2o.it,deejay.it,capital.it#@##iubenda-cs-banner
```

Comment:
I removed `###iubenda-cs-banner` and exclusions. We will use the domain specific rule.